### PR TITLE
FE-60: Simple enterprise icons for different text levels

### PR DESCRIPTION
**Pull request by zhengmar3**
### Overview
The Corda 5 GA release will include some functionality that is "Enterprise" only. Unlike Corda 4 documentation, this will not be documented as two distinct products but rather one set of docs with the "Enterprise-only" content included in the relevant area of the "Corda 5" documentation.

"The suggestion is to have something like the following symbol, with hover text containing a one liner explaining it, but also with a ‘more’ link to an introduction section that explains it in detail.:

Jira: https://r3-cev.atlassian.net/browse/FE-60 

### Screenshot

For h2 heading:

`## Layered Architecture {{<enterprise-h2-icon>}}`
image

For h3 heading:

`### P2P Layer {{<enterprise-h3-icon>}}`
image

For generic text:
`The Corda platform {{<enterprise-generic-icon>}} is a layered toolbox`
image

Tooltip:

image

### CODE REVIEW

**Review by colin-hillis — APPROVED** [review]
Thank you Zheng for sticking with this through all the iterations. It is much appreciated, plus it looks good. 👍

File: themes/doks/layouts/shortcodes/enterprise-h2-icon.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    margin-top: 2px;
+    padding-left: 10px !important;
+    padding-right: 10px !important;
+    padding-top: 4px !important;
+    padding-bottom: 4px !important;
+    border-color: #D9DBE9;
+  }
+
+</style>
+<a title="This feature is only available in the Enterprise edition of Corda. <More info. link>" href="#detailedlinkgoeshere" class="enterprise-h2-icon">E</a>
```
Comment (nadinequinr3) [inline themes/doks/layouts/shortcodes/enterprise-h2-icon.html:20]:
```suggestion
<a title="This feature is only available in the Enterprise edition of Corda." href="#detailedlinkgoeshere" class="enterprise-h2-icon">E</a>
```

**Review by nadinequinr3 — DISMISSED** [review]
Looks good, thanks @zhengmar3

File: themes/doks/layouts/shortcodes/enterprise-h3-icon.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    border-radius: 6px;
+    padding-left: 10px !important;
+    padding-right: 10px !important;
+    padding-top: 4px !important;
+    padding-bottom: 4px !important;
+    border-color: #D9DBE9;
+  }
+
+</style>
+<a title="This feature is only available in the Enterprise edition of Corda. <More info. link>" href="#detailedlinkgoeshere" class="enterprise-h3-icon">E</a>
```
Comment (nadinequinr3) [inline themes/doks/layouts/shortcodes/enterprise-h3-icon.html:19]:
```suggestion
<a title="This feature is only available in the Enterprise edition of Corda." href="#detailedlinkgoeshere" class="enterprise-h3-icon">E</a>
```

**Review by nadinequinr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
@zhengmar3 I think we should have an icon representing Enterprise that has more than 1 letter E.
Maybe 'Ent' 🤔

**Comment by colin-hillis** [comment]
> @zhengmar3 I think we should have an icon representing Enterprise that has more than 1 letter E. Maybe 'Ent' 🤔

Happy to proceed with E. There has been a tremendous amount of back and forth and talk around what we should and shouldn't do with this, but there is fairly strong support for keeping it low key and less impactful on screen. The hover text and link for more info should cover it. I will write a section to explain it more fully shortly.

There is another icon to be added beside headings too, so we don't want it starting to dominate too much.

**Pull request merged**